cli/common.go

```diff
@@ -62,7 +93,7 @@ func createRequest(dirName string) (controller.Request, error) {
 		req.Units = append(req.Units, controller.Unit{Name: name, Content: content})
 	}
 
-	sliceIDs, err := createSliceIDs(dirName)
+	sliceIDs, err := createSliceIDs(slices)
```

Comment:
mh, its probably indeed useless, but to me it feels more correct. Lets assume this is the call:

```
err := controller.Submit(controller.SubmitRequest{
  SliceIDs: []string{"1", "2", "3"},
  Units: []controller.Unit{
     controller.Unit{"appd-main@.service", "..."},
     controller.Unit{"appd-lb-register@.service", "..."},
     controller.Unit{"appd-ambassador@.service", "..."},
  }
})
```

to my eye it just feels weird to not have a `Name` field in there. Also I would probably expect the controller to double check the provided arguments (do the units start with `Name`, if we have slices, do the units contain a `@` etc.), so then its needed. 

But again: Currently its not needed and this is purely my mind wishing this for sugar reasons.